src/aat/resources/features/F-040/F-040.feature

```diff
@@ -0,0 +1,21 @@
+@F-040
+Feature: F-040: Get Case for Case worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-576
+  Scenario: must return successfully all and only the respective fields with READ access for Solicitors
+    Given a user with [an active Solicitor profile in CCD with a specific variation of ACLs on a case type]
+    And a user with [another active solicitor profile with different variation of ACLs on the same case type]
```

Comment:
please add "And   a user with [an active Solicitor profile having full permissions on the same case type]" as we are using 3 solicitor